### PR TITLE
SUBMARINE-1207. Move server-core/workbench/entity to server-api/workbench

### PR DESCRIPTION
**Pull request by Ray02250418**
### What is this PR for?
Move files under submarine-server/server-core/src/main/java/org/apache/submarine/server/workbench/entity to submarine-server/server-api/src/main/java/org/apache/submarine/server/api/workbench.

### What type of PR is it?
Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-1207?filter=allopenissues

### Screenshots (if appropriate)
No

### Questions:
* Do the license files need updating? /No
* Are there breaking changes for older versions? No
* Does this need new documentation? No

### CODE REVIEW

File: submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/SysDictItemRestApiTest.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.submarine.server.workbench.rest;
+package org.apache.submarine.server.rest.workbecnh;
```
Comment (jeff-901) [inline submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/SysDictItemRestApiTest.java:19]: typo workbench.

File: submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/CommonDataTest.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.submarine.server.workbench.rest;
+package org.apache.submarine.server.rest.workbecnh;
```
Comment (jeff-901) [inline submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/CommonDataTest.java:19]: typo workbench for all *Test.java

Comment (jeff-901) [inline submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/CommonDataTest.java:19]: typo workbench for all *Test.java

File: submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/CommonDataTest.java [hunk 3]
```diff
@@ -29,6 +29,10 @@
 import org.apache.submarine.server.workbench.database.service.SysUserService;
 import org.apache.submarine.server.response.JsonResponse;
 import org.apache.submarine.server.response.JsonResponse.ListResult;
+import org.apache.submarine.server.rest.workbench.SysDeptRestApi;
```
Comment (jeff-901) [inline submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/CommonDataTest.java:32]: @Ray02250418  After you correct the package name, you don't have to import those classes.

Comment (jeff-901) [inline submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/workbecnh/CommonDataTest.java:32]: @Ray02250418  After you correct the package name, you don't have to import those classes.

**Review by jeff-901 — CHANGES_REQUESTED** [review]

**Comment by MortalHappiness** [comment]
The PR title should be `SUBMARINE-1207` instead of `Submarine 1207`.

**Comment by xunliu** [comment]
hi @Ray02250418 
Please change PR title to `SUBMARINE-1207. Move ...`

**Comment by Ray02250418** [comment]
@jeff-901 Please help me review the code.

**Review by jeff-901 — APPROVED** [review]
LGTM

**Pull request closed without merging**